<!doctype html>
<html lang="en">

<head>
  <!-- Pie Chart by Plotly -->
  <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
  <title>Marco Notaro Interest</title>
  <meta charset="UTF-8">
  <meta name="author" content="Marco Notaro">
  <meta name="description" content="Marco Notaro WebSite">
  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- to make the site responsive -->
  <meta name="keywords" content="bioinformatics, computational biology, machine learning, ngs, multi-omics data">
  <link rel="shortcut icon" href="img/wolf.gif">
  <link rel="stylesheet" type="text/css" href="css/css.css">
</head>

<body>
  <div id="container">
    <div id="header">
      <a href="index.html" title="Marco Notaro"><img id="logo" src="img/marcologo.png" alt="Marco Notaro" height="50"></a>
    </div>

    <div id="navigationbar">
      <div id="navigation">
        <ul class="menubar">
          <li><b><a class="menubar" href="index.html">Home</a></b></li>
          <li><b><a class="menubar" href="interest.html">Interest</a></b></li>
          <li><b><a class="menubar" href="teaching.html">Teaching</a></b></li>
          <li><b><a class="menubar" href="publications.html">Publications</a></b></li>
          <li><b><a class="menubar" href="software.html">Software</a></b></li>
          <li><b><a class="menubar" href="map.html">Visited Places</a></b></li>
        </ul>
      </div>
    </div>

    <div class="bodycontainer">

    <h3>Overview</h3>
      <p>
        Thanks to high-throughput biotechnologies, we are assisting to a massive stacking of genomic, transcriptomic, proteomic and
        metabolomic data. Molecular biology is nowadays turning into a <i>"data-rich"</i> science.
        This unprecedented richness of raw data is meaningless until we cannot extract the interesting and useful
        "<i>needles</i>" of information from the big "<i>haystack</i>" of available data.
        This allow to us to have at the same time both a "<i>birdâ€™s eye</i>" and a "<i>magnified</i>" overview on a biological
        sample from a wide range of medically relevant disorders. <i>"The answer is there, blowing in the bulk"</i> (thanks Zimmerman &#128515;).
        However, this huge amount of raw data is of little use without groundbreaking bioinformatics and biostatistics methodologies
        able to disentangle, process, analyze and interpret them.
      </p>
      <p>
        In my work, I strike a balance between two ways of doing science. On one hand I focus on the development and application of statistical and machine learning approaches
        for analyzing high-throughput and heterogeneous biomedical data. On the other hand, I package such approaches into modular, scalable and user-friendly
        tools, in order to share them with the scientific community.
      </p>

    <h3>Area of Interest</h3>
      <ol class="interest">
        <li class="interest">
          <b>BIO-MOLECULAR NETWORKS</b><br> analysis and construction of complex biomolecular networks by making use of database such as
          <a href="http://string-db.org/">STRING</a>, <a href="https://thebiogrid.org/">BioGRID</a> and <a href="http://www.uniprot.org/">UniProt</a>;
        </li>
        <li class="interest">
          <b>FLAT LEARNING METHODS</b><br> application of both supervised and semi-supervised learning methods to make "flat" predictions,
          i.e. predictions that do not consider the hierarchical structure of the label space;
        </li>
        <li class="interest">
          <b>OUTPUT STRUCTURED LEARNING</b><br> design and software implementation of hierarchical ensemble methods (HEMs) for the
          prediction of protein function (an open fundamental challenge in molecular biology) and for the prediction of human gene-abnormal
          phenotype associations (a crucial step to discover novel genes associated to Mendelian diseases).
          The HEMs' predictions are consistent with the ontology predictions, because they always obey to the "<i>true path rule</i>",
          the logical and biological rule that governs the internal coherence of the bio-ontologies, such as the Gene Ontology
          (<a href="http://www.geneontology.org/">GO</a>) and the Human Phenotype Ontology (<a href="http://human-phenotype-ontology.github.io/">HPO</a>).
        </li>
        <li class="interest">
          <b>NEXT GENERATION SEQUENCING</b><br> application of computational tools for the analysis of genetic variants in sequencing data, such as copy number,
          single nucleotide variants, INDELs and the analysis of next-generation RNA/ATAC/ChIP-Seq data sequencing data.
        </li>
        <li class="interest">
          <b>MULTI-OMICS DATA</b><br> disentangling multi-omics data through statistical approaches and building up shiny applications to support clients
          in the process of hypotheses generation.
        </li>
      </ol>
    </div> <!-- end bodycontainer -->
    <div id="footer"></div>
  </div>  <!-- end container -->
</body>
</html>
