<!doctype html>
<html lang="en">

<head>
  <!-- Pie Chart by Plotly -->
  <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
  <title>Marco Notaro Interest</title>
  <meta charset="UTF-8">
  <meta name="author" content="Marco Notaro">
  <meta name="description" content="Marco Notaro WebSite">
  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- to make the site responsive -->
  <meta name="keywords" content="bioinformatics, computational biology, machine learning, ngs, multi-omics data">
  <link rel="shortcut icon" href="img/wolf.gif">
  <link rel="stylesheet" type="text/css" href="css/css.css">
</head>

<body>
  <div id="container">
    <div id="header">
      <a href="index.html" title="Marco Notaro"><img id="logo" src="img/marcologo.png" alt="Marco Notaro" height="50"></a>
    </div>

    <div id="navigationbar">
      <div id="navigation">
        <ul class="menubar">
          <li><b><a class="menubar" href="index.html">Home</a></b></li>
          <li><b><a class="menubar" href="interest.html">Interest</a></b></li>
          <li><b><a class="menubar" href="teaching.html">Teaching</a></b></li>
          <li><b><a class="menubar" href="publications.html">Publications</a></b></li>
          <li><b><a class="menubar" href="software.html">Software</a></b></li>
          <li><b><a class="menubar" href="map.html">Visited Places</a></b></li>
        </ul>
      </div>
    </div>

    <div class="bodycontainer">

    <h3>Overview</h3>
      <p>
        Thanks to high-throughput biotechnologies, we are assisting a massive stacking of genomic, transcriptomic, proteomic and
        metabolomic data. Molecular biology is nowadays turning into a <i>"data-rich"</i> science.
        This unprecedented richness of raw data is meaningless until we cannot extract the interesting and useful
        "<i>needles</i>" of information from the big "<i>haystack</i>" of available data.
        This allows us to have at the same time both a "<i>birdâ€™s eye</i>" and a "<i>magnified</i>" overview on a biological
        sample from a wide range of medically relevant disorders. <i>"The answer is there, blowing in the bulk"</i> (thanks Zimmerman &#128515;).
        However, this huge amount of raw data is of little use without groundbreaking bioinformatics and biostatistics methodologies
        able to disentangle, process, analyze and interpret them.
      </p>
      <p>
        In my work, I strike a balance between two ways of doing science. On one hand I focus on the development and application of statistical and machine learning approaches
        for analyzing high-throughput and heterogeneous biomedical data. On the other hand, I package such approaches into modular, scalable and user-friendly
        tools, in order to share them with the scientific community.
      </p>

    <h3>Skills by Interest Area</h3>
      <ol class="interest">
        <li class="interest">
          <b>Next Generation Sequencing / Multi-Omics Data</b><br>
          <ul class="interest">
            <li> Developing tailored analytics solutions for NGS datasets (WES, bulk RNAseq, scRNAseq, ChIP-Seq, CUT&RUN, CRISPR-screen)
            <li> Developing scalable and reproducible pipelines for the analysis of NGS datasets
            <li> Performing variant calling analysis to detect point and structural mutations
            <li> Performing clustering analysis to stratify patients on the basis of gene/protein expression profiles
            <li> Performing differential gene/protein expression and pathway enrichment analysis
            <li> Performing differential binding/accessibility analysis with ChIP-Seq/ATAC-Seq data
            <li> Quantifying chromatin signal metaprofiles
            <li> Programmatic configuration and browsing of genomic data via UCSC track hubs
            <li> Developing Shiny web applications to interactively explore multi-omics data <!-- to facilitate the process of data exploration and hypothesis generation -->
            <!-- <li> Disentangling, integrating and visualizing multi-omics data through statistical approaches -->
            <!-- <li> Analyzing LC-MS proteomics data -->
          </ul>
        </li>
        <li class="interest">
          <b>Machine Learning</b><br>
          <ul class="interest">
            <li> Developing ensemble methods for the functional annotation of gene products
            <li> Implementing workflow for executing machine learning experiments (hold-out, k-fold cross-validation)
            <li> Applying machine learning methods for the prediction of human gene-abnormal phenotype associations and for the prediction of protein function
            <li> Implementing ensemble methods in the R/CRAN software library (<a href="https://hemdag.readthedocs.io" target="_blank">HEMDAG</a>)
            <li> Implementing a Perl module for handling bio-ontologies and gene annotation files (<a href="https://obogaf-parser.readthedocs.io" target="_blank">obogaf-parser</a>)
            <li> Developing an integrated R/Perl pipeline to build datasets for machine learning experiments (<a href="https://github.com/marconotaro/godata-pipe" target="_blank">godata-pipe</a>)
          </ul>
        </li>
        <li class="interest">
          <b>Bio-molecular networks</b><br>
          <ul class="interest">
            <li> Wrangling and analysis of biomolecular networks
            <li> Wrangling and analysis of heterogeneous ontology and interaction graphs (protein-coding and non-coding genes)
            <!-- <li> Building complex biomolecular networks -->
            <!-- <li> Building heterogeneous ontology graphs -->
            <!-- <li> Building heterogeneous interaction graphs of protein-coding and non-coding genes -->
          </ul>
        </li>
      </ol>
    </div> <!-- end bodycontainer -->
    <div id="footer"></div>
  </div>  <!-- end container -->
</body>
</html>
