<!doctype html>
<html lang="en">

<head>
    <!-- Google Analytics Tracking Code -->
    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
        ga('create', 'UA-90875446-1', 'auto');
        ga('send', 'pageview');
    </script>
    <!-- Google Analytics Tracking Code -->
    <title>Marco Notaro Papers</title>
    <meta charset="UTF-8">
    <meta name="author" content="Marco Notaro">
    <meta name="description" content="Marco Notaro WebSite">
    <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- to make the site responsive -->
    <meta name="keywords" content="papers, publications, journal, conference, theses, book chapters, peer-reviewed, bioinformatics, 
        computational biology, machine learning, hierarchical ensemble methods, biological database, biological networks">
    <link rel="shortcut icon" href="img/wolf.gif">
    <link rel="stylesheet" type="text/css" href="css/css.css"> 
    <script src="js/reverse.ordering.js"></script>
</head>

<body>
    <div id="container">
        <div id="header">
            <a href="index.html" title="Marco Notaro"><img id="logo" src="img/marcologo.png" alt="Marco Notaro" height="50"></a>     
        </div>

        <div id="navigationbar">    
            <div id="navigation">
                <ul class="menubar">    
                    <li><b><a class="menubar" href="index.html">Home</a></b></li>
                    <li><b><a class="menubar" href="research.html">Research</a></b></li>
                    <li><b><a class="menubar" href="teaching.html">Teaching</a></b></li>
                    <li><b><a class="menubar" href="publications.html">Publications</a></b></li>
                    <li><b><a class="menubar" href="software.html">Software</a></b></li>
                    <li><b><a class="menubar" href="contact.html">Contact</a></b></li>
                    <li><b><a class="menubar" href="visitedplaces.html">Visited Places</a></b></li>
                </ul>
            </div>      
        </div>  
        
        <div class="bodycontainer">

            <h3> International Peer-Reviewed Journals </h3>
            <ol class="reversed">
                <li class="papers">
                    <p> 
                        <b>M. Notaro</b>, M. Frasca, A. Petrini, G. Valentini. HEMDAG: a scalable and flexible hierarchical ensemble-based tool
                        to improve Gene Ontology term predictions; 2019. <b>Note</b>: <i>manuscript in preparation</i>.
                    </p>
                </li>
                <li class="papers">
                    <p> 
                        N. Zhou, Y. Jiang, ..., <b>M. Notaro</b>, ..., P. Radivojac, I. Friedberg. The CAFA challenge reports improved protein 
                        function prediction and new functional annotations for hundreds of genes through experimental screens. 
                        <i>bioRxiv</i>. 2019. Cold Spring Harbor Laboratory.
                        (
                            <a target="_blank" href="https://www.biorxiv.org/node/752384.external-links.html">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="https://www.biorxiv.org/content/biorxiv/early/2019/05/29/653105.full.pdf">
                            <img src="img/pdf-resize.png" alt="pdf_icon_link"></a>
                        )
                    </p>
                </li>
                <li class="papers">
                    <p> 
                        P. Perlasca, M. Frasca, C.T. Ba, <b>M. Notaro</b>, A. Petrini, G. Grossi, J. Gliozzo, G. Valentini and M. Mesiti. 
                        <i>Unipred-web: a web tool for the integration and visualization of biomolecular networks for protein function prediction</i>. 
                        <i>BMC Bioinformatics</i> 2019. 20(1):422:1–422:19.

                    </p>
                </li>
                <li class="papers">
                    <p> 
                        M. Frasca, G. Grossi, J. Gliozzo, M. Mesiti, <b>M. Notaro</b>, P. Perlasca, A. Petrini and G. Valentini,
                        <i>A GPU-based algorithm for fast node label learning in large and unbalanced biomolecular networks</i>,
                        <i>BMC Bioinformatics</i> 2018. 19-S(10):269–280.
                        (
                            <a target="_blank" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2301-4">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2301-4">
                            <img src="img/pdf-resize.png" alt="pdf_icon_link"></a>
                        )
                    </li>   
                <li class="papers">
                    <p> 
                        <b>M. Notaro</b>, M. Schubach, P. Robinson, and G. Valentini,
                        <i>Prediction of Human Phenotype Ontology terms by means of Hierarchical Ensemble methods</i>,
                        <i>BMC Bioinformatics</i>, 2017. 18(1):449:1-449:18.
                        (
                            <a target="_blank" href="https://doi.org/10.1186/s12859-017-1854-y">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1854-y">
                            <img src="img/pdf-resize.png" alt="pdf_icon_link"></a>
                        )           
                        <b>Note</b>: <i>awarded by the International Medical Informatics Association (IMIA) as one of the five best Knowledge Representation and Management (KRM) papers of 2017 in the field of Medical Informatics</i>            
                        (
                            <a target="_blank" href="https://www.thieme-connect.com/products/ejournals/abstract/10.1055/s-0038-1667078">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="https://medium.com/@MonarchInit/hpo-paper-chosen-as-a-top-knowledge-representation-and-management-publication-f4a207e687fe">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="http://www.unimi.it/lastatalenews/articolo-anacletolab-cinque-migliori-dellanno-per-limia">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>
                        )
                    </p>
                </li>
            </ol>     

            <h3> Proceedings of International Conferences and Peer-Reviewed Book Chapters </h3>
            <ol class="reversed">
                <li class="papers">
                    <p> 
                        <b>M. Notaro</b>, M. Schubach, P.N. Robinson, G. Valentini,
                        <i>Ensembling Descendant Term Classifiers to Improve Gene - Abnormal Phenotype Predictions</i>,
                        In: Bartoletti M. et al. (eds) Computational Intelligence Methods for Bioinformatics and Biostatistics. 
                        CIBB 2017. Lecture Notes in Computer Science, vol 10834. Springer, Cham. 
                        (
                            <a target="_blank" href="https://link.springer.com/chapter/10.1007%2F978-3-030-14160-8_8">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="papers/conferences/paper3_lnbi.pdf">
                            <img src="img/pdf-resize.png" alt="pdf_icon_link"></a>
                        )
                    </p>
                </li>
                <li class="papers">
                    <p> 
                        M. Frasca, J.F. Fontaine, G. Valentini, M. Mesiti, <b>M. Notaro </b>, D. Malchiodi and M.A. Andrade-Navarro 
                        <i>Disease Genes must Guide Data Source Integration in the Gene Prioritization Process</i>,
                        In: Bartoletti M. et al. (eds) Computational Intelligence Methods for Bioinformatics and Biostatistics. 
                        CIBB 2017. Lecture Notes in Computer Science, vol 10834. Springer, Cham.
                        (
                            <a target="_blank" href="https://link.springer.com/chapter/10.1007%2F978-3-030-14160-8_7">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="papers/conferences/paper2_lnbi.pdf">
                            <img src="img/pdf-resize.png" alt="pdf_icon_link"></a>
                        )
                    </p>
                </li>
                <li class="papers">
                    <p> 
                        P.N. Robinson, M. Frasca, S. K&ouml;hler, <b>M. Notaro</b>, M. Re, G. Valentini, 
                        <i> A Hierarchical Ensemble Method for DAG-Structured Taxonomies</i>,
                        In: Schwenker F., Roli F., Kittler J. (eds) Multiple Classifier Systems. MCS 2015. 
                        Lecture Notes in Computer Science, vol 9132. Springer, Cham.
                        (
                            <a target="_blank" href="http://link.springer.com/chapter/10.1007/978-3-319-20248-8_2">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="papers/conferences/paper1_mcs.pdf">
                            <img src="img/pdf-resize.png" alt="pdf_icon_link"></a>
                        )
                    </p>
                </li>
                <li class="papers">
                    <p> 
                        G. Valentini, S. K&ouml;hler, M. Re, <b>M. Notaro</b>, P.N. Robinson, 
                        <i>Prediction of Human Gene-Phenotype Associations by Exploiting the Hierarchical 
                        Structure of the Human Phenotype Ontology</i>, In: Ortu&#241;o F., Rojas I. (eds) Bioinformatics and Biomedical Engineering. 
                        IWBBIO 2015. Lecture Notes in Computer Science, vol 9043. Springer, Cham.                
                        (
                            <a target="_blank" href="http://link.springer.com/chapter/10.1007/978-3-319-16483-0_7">
                            <img src="img/net-resize.png" alt="net_icon_link"></a>,
                            <a target="_blank" href="papers/conferences/paper0_iwbbio.pdf">
                            <img src="img/pdf-resize.png" alt="pdf_icon_link"></a>
                        )
                    </p>
                </li>
            </ol>   

            <h3>Theses</h3>
            <ol class="reversed">
                <li class="papers">
                    <p> 
                        <b>Marco Notaro</b>, <i> Hierarchical Ensemble Methods for Ontology-based Predictions in Computational Biology</i>, 
                        Ph.D. in Computer Science, Milan University, February 2019
                        ( <a target="_blank" href="papers/thesis/phd_thesis.pdf"><img src="img/pdf-resize.png" alt="pdf_icon_link"></a>, English)
                    </p>                    
                </li>
                <li class="papers">
                    <p> 
                        <b>Marco Notaro</b>, <i> Hierarchical Ensemble Methods for Biontologies Structured as Directed Acyclic Graphs</i>,
                         M.Sc. in Molecular Biotechnologies and Bioinformatics, Milan University, February 2015
                        ( <a target="_blank" href="papers/thesis/mcs_thesis.pdf"><img src="img/pdf-resize.png" alt="pdf_icon_link"></a>, Italian)
                    </p>                    
                </li>
                <li class="papers">
                    <p> 
                        <b>Marco Notaro</b>, <i>High Resolution Maps of Methylation and Acetylation of Histones in the Human Genome</i>, 
                        B.Sc. in Industrial and Environmental Biotechnology, Milan University, October 2012
                        ( <a target="_blank" href="papers/thesis/bsc_thesis.pdf"><img src="img/pdf-resize.png" alt="pdf_icon_link"></a>, Italian)
                    </p>                    
                </li>
            </ol>
            
            <h3>Elsewhere</h3>
                <p class="elsewhere">
                    You can also find a full list of my publications (and other stuff) on the following pages:
                </p>
                <ol>
                    <li class="elsewhere">
                        <p>
                            <a href="https://www.scopus.com/authid/detail.uri?authorId=56728073200" target="_blank">Scopus</a>
                        </p>
                    </li>
                    <li class="elsewhere">
                        <p>
                            <a href="https://orcid.org/0000-0003-4309-2200" target="_blank">ORCID</a>
                        </p>
                    </li>
                    <li class="elsewhere">
                        <p>
                            <a href="http://dblp.uni-trier.de/pers/hd/n/Notaro:Marco" target="_blank">DBLP</a>
                        </p>
                    </li>
                    <li class="elsewhere">
                        <p>
                            <a href="https://air.unimi.it/browse?type=author&order=ASC&rpp=20&authority=rp38272#.WKXM31dk_rc" target="_blank">IRIS-AIR@UNIMI</a>
                        </p>
                    </li>
                    <li class="elsewhere">
                        <p>
                            <a href="https://scholar.google.it/citations?user=BIl3KZwAAAAJ&hl=en" target="_blank">Google Scholar</a>
                        </p>
                    </li>           
                </ol>
        </div> <!-- end bodycontainer --> 
        <div id="footer"></div>
        <script> reverse_lists(); </script>
    </div>  <!-- end container --> 
</body>
</html>

