<!doctype html>
<html lang="en">

<head>
    <!-- Google Analytics Tracking Code -->
    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
        ga('create', 'UA-90875446-1', 'auto');
        ga('send', 'pageview');
    </script>
    <!-- Google Analytics Tracking Code -->
    <!-- Pie Chart by Plotly -->
    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
    <title>Marco Notaro Research</title>
    <meta charset="UTF-8">
    <meta name="author" content="Marco Notaro">
    <meta name="description" content="Marco Notaro WebSite">
    <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- to make the site responsive -->
    <meta name="keywords" content="research activity, bioinformatics, computational biology, machine learning, hierarchical ensemble methods,
        biological database, biological networks, R, Perl, Python, software libraries">
    <link rel="shortcut icon" href="img/wolf.gif">
    <link rel="stylesheet" type="text/css" href="css/css.css">
</head>

<body>
    <div id="container">
        <div id="header">
            <a href="index.html" title="Marco Notaro"><img id="logo" src="img/marcologo.png" alt="Marco Notaro" height="50"></a>
        </div>

        <div id="navigationbar">
            <div id="navigation">
                <ul class="menubar">
                    <li><b><a class="menubar" href="index.html">Home</a></b></li>
                    <li><b><a class="menubar" href="research.html">Research</a></b></li>
                    <li><b><a class="menubar" href="teaching.html">Teaching</a></b></li>
                    <li><b><a class="menubar" href="publications.html">Publications</a></b></li>
                    <li><b><a class="menubar" href="software.html">Software</a></b></li>
                    <li><b><a class="menubar" href="contact.html">Contact</a></b></li>
                    <li><b><a class="menubar" href="citymap.html">Visited Places</a></b></li>
                </ul>
            </div>
        </div>

        <div class="bodycontainer">

        <h3>Overview</h3>
            <p>
                Thanks to high-throughput biotechnologies, we are assisting to a massive stacking of genomic, transcriptomic, proteomic and
                metabolomic data. Molecular biology is nowadays turning into a <i>"data-rich"</i> science.
                This unprecedented richness of raw data is meaningless until we cannot extract the interesting and useful
                "<i>needles</i>" of information from the big "<i>haystack</i>" of available data.
                This allow to us to have at the same time both a "<i>birdâ€™s eye</i>" and a "<i>magnified</i>" overview on a biological
                sample from a wide range of medically relevant disorders. <i>"The answer is there, blowing in the bulk"</i> (thanks Zimmerman &#128515;).
                However, this huge amount of raw data is of little use without groundbreaking bioinformatics and biostatistics methodologies
                able to disentangle, process, analyze and interpret them.
            </p>
            <p>
                In my research, I would like to strike a balance between two ways of doing science. On one hand I would like to focus on the
                design, development and application of novel statistical and machine learning approaches for analyzing heterogeneous and high-dimensional
                biomedical data. On the other hand, I would like to package such approaches into modular, scalable and user-friendly software tools,
                in order to make them available to the scientific community at large (have a look at my <a href="software.html"> Software</a> page).
            </p>

        <h3>Bioinformatics vs Computational Biology</h3>
            <p>
                Even if Bioinformatics and Computational Biology are often considered synonymous, I believe they are conceptually
                separated and it is worth distinguishing them.
            </p>
            <ul class="research">
                <li class="research">
                    <b>Computational Biology</b>: is the study of Biology using computational techniques. The main goal of a
                    computational biologist is to make new insights about Biology and living system.
                    Then Computational Biology is about <b>Science</b>.
                </li>
                <li class="research">
                    <b>Bioinformatics</b>: is about the creation of new algorithms able to solve problems. The main goal of a
                    bioinformatician is to build tools that can work on biological, medical and pharmaceutical data.
                    Then Bioinformatics is about <b>Computer Science</b>.
                </li>
            </ul>

            <p>
                When I <i>design</i> new methodologies I am involved in a Computer Science activity.
                I design algorithms not to solve just a single specific problem, but a class of similar problems in Medicine or Molecular Biology.
            </p>

            <p>
                When I use my software (or those of others) to <i>answer</i> biomedical questions, I am doing science and
                I am making inferences in the field of Molecular Biology or Medicine.
            </p>

            <p>
                Anyway, despite the difference between Bioinformatics and Computational Biology sounds apparently easy and clear,
                I believe that dealing with complex computational issues without losing sight of the medical/biological point of view,
                is simultaneously challenging and captivating!!
            </p>

        <h3>Research Interests</h3>
            <p>
                During my PhD, I focused on the analysis and construction of complex biomolecular networks and on design and implementation of output-structured learning algorithms
                for gene/protein function prediction and for the discovery of novel associations between human gene and abnormal phenotype.
                More in general, my research lines can be grouped as follow:
            </p>
            <ol class="research">
                <li class="research">
                    <b>BIO-MOLECULAR NETWORKS</b><br> analysis and construction of complex biomolecular networks by making use of database such as
                    <a href="http://string-db.org/">STRING</a>, <a href="https://thebiogrid.org/">BioGRID</a> and <a href="http://www.uniprot.org/">UniProt</a>;
                </li>
                <li class="research">
                    <b>FLAT LEARNING METHODS</b><br> application of both supervised and semi-supervised learning methods to make "flat" predictions,
                    i.e. predictions that do not consider the hierarchical structure of the label space;
                </li>
                <li class="research">
                    <b>OUTPUT STRUCTURED LEARNING</b><br> design and software implementation of hierarchical ensemble methods (HEMs) for the
                    prediction of protein function (an open fundamental challenge in molecular biology) and for the prediction of human gene-abnormal
                    phenotype associations (a crucial step to discover novel genes associated to Mendelian diseases).
                    The HEMs' predictions are consistent with the ontology predictions, because they always obey to the "<i>true path rule</i>",
                    the logical and biological rule that governs the internal coherence of the bio-ontologies, such as the Gene Ontology
                    (<a href="http://www.geneontology.org/">GO</a>) and the Human Phenotype Ontology (<a href="http://human-phenotype-ontology.github.io/">HPO</a>).
                </li>
                <li class="research">
                    <b>NEXT GENERATION SEQUENCING</b><br> application of computational tools for the analysis of genetic variants in sequencing data, such as copy number,
                    single nucleotide variants, INDELs and the analysis of next-generation ChIP/RNA-Seq data sequencing data.
                </li>
            </ol>

            <h3>Interdisciplinary Research</h3>
            The pie chart displayed below groups my publications by subject area.
            From this graph it is clear how my research interests are equally spanned over the areas of Computer Science, Mathematics and Biomolecular
            Science (Biochemistry, Genetics and Molecular Biology).
            The pie graph shown below was created by <a target="_blank" href="https://plot.ly/">Plotly</a>, whereas the partition of publications by subject area is taken from <a target="_blank" href="https://www.scopus.com/hirsch/author.uri?accessor=authorProfile&auidList=56728073200&authorName=Notaro%2c+Marco&origin=AuthorProfile&txGid=f3e62d6905193fbc2446abc5b36bf94f">Scopus - Documents by subject area</a>.

            <div id="plotly"></div>
            <script>
                if (matchMedia){
                    const mq = window.matchMedia("(max-width: 960px)");
                    mq.addListener(WidthChange);
                    WidthChange(mq);
                }
                function WidthChange(mq){
                    if(mq.matches){
                        var data = [{
                            values: [7,7,4,1],
                            labels: [ 'Computer Science',
                                      'Mathematics',
                                      'Biochemistry, Genetics and Molecular Biology',
                                      'Agricultural and Biological Sciences'
                                    ],
                            type: 'pie',
                            hoverinfo: "label+percent",
                            textinfo: "label+percent",
                            textposition: "inside",
                            insidetextorientation: "auto",
                            automargin: true
                        }];

                        var layout = {
                        automargin: true,
                            showlegend: true,
                            title: {
                                text:'<b>Documents by subject area</b>',
                                font: {
                                    family: 'Courier New, monospace',
                                    size: 13,
                                }
                            },
                            legend: {
                                "orientation": "h",
                                font: {
                                    family: 'sans-serif',
                                    siee: 2
                                }
                            },
                            margin: { "t": 50, "b": 0, "l": 0, "r": 0 },
                            font: {
                                family: 'sans-serif',
                                size: 12
                            }
                        };

                        var config = {
                            responsive: true,
                            displayModeBar: false,
                            // modeBarButtonsToRemove: ['toImage', 'hoverClosestPie'],
                            // displaylogo: true
                        }

                        Plotly.newPlot('plotly', data, layout, config);
                    }else{
                        var data = [{
                            values: [7,7,4,1],
                            labels: [ 'Computer Science',
                                      'Mathematics',
                                      'Biochemistry, Genetics and Molecular Biology',
                                      'Agricultural and Biological Sciences'
                                    ],
                            type: 'pie',
                            hoverinfo: "label+percent",
                            textinfo: "label+percent",
                            textposition: "outside",
                            automargin: true
                        }];

                        var layout = {
                        automargin: true,
                            showlegend: true,
                            title: {
                                text:'<b>Documents by subject area</b>',
                                font: {
                                    family: 'Courier New, monospace',
                                    size: 16,
                                }
                            },
                            legend: {
                                "orientation": "h",
                                font: {
                                    family: 'sans-serif',
                                }
                            },
                            margin: { "t": 80, "b": 80, "l": 0, "r": 0 },
                            font: {
                                family: 'sans-serif',
                                size: 12
                            }
                        };

                        var config = {
                            responsive: true,
                            displayModeBar: false,
                            // modeBarButtonsToRemove: ['toImage', 'hoverClosestPie'],
                            // displaylogo: true
                        }

                        Plotly.newPlot('plotly', data, layout, config);
                    }
                }
            </script>
        </div> <!-- end bodycontainer -->
        <div id="footer"></div>
    </div>  <!-- end container -->
</body>
</html>

